src/components/home/Spine.tsx

```diff
@@ -1,31 +1,31 @@
-import type { Entity} from "oasis-engine";
-import { Camera, Vector3, WebGLEngine, Vector4 } from "oasis-engine";
-import { SpineAnimation } from "@oasis-engine/engine-spine";
-import { OrbitControl } from "@oasis-engine/controls";
-import React, { useEffect } from "react";
+import type { Entity } from 'oasis-engine';
+import { Camera, Vector3, WebGLEngine, Vector4 } from 'oasis-engine';
+import { SpineAnimation } from '@oasis-engine/engine-spine';
+import { OrbitControl } from '@oasis-engine/controls';
+import React, { useEffect } from 'react';
 
-export default function PBRHelmet () {
+export default function PBRHelmet() {
   useEffect(() => {
     const engine = init();
 
     return () => {
       engine.destroy();
-    }
+    };
   });
 
-  return <canvas id="canvas-spine" style={{width: '300px', height: '400px'}}/>
+  return <canvas id="canvas-spine" style={{ width: '300px', height: '400px' }} />;
 }
 
-function init (): WebGLEngine {
-  const engine = new WebGLEngine("canvas-spine");
+function init(): WebGLEngine {
+  const engine = new WebGLEngine('canvas-spine');
   engine.canvas.resizeByClientSize();
   const scene = engine.sceneManager.activeScene;
   const rootEntity = scene.createRootEntity();
 
   // camera
-  const cameraEntity = rootEntity.createChild("camera_node");
-  const camera = cameraEntity.addComponent(Camera);
-  camera.backgroundColor = new Vector4(51/255, 51/255, 51/255, 1);
+  const cameraEntity = rootEntity.createChild('camera_node');
+  cameraEntity.addComponent(Camera);
+  scene.background.solidColor.setValue(51 / 255, 51 / 255, 51 / 255, 1);
```

Comment:
The display of background color with the same value is different from v0.3. See the 2D demo in homepage of site.